raster/raster-layer-file/README.md

```diff
@@ -1,21 +1,32 @@
-# Raster Layer File
+# Raster layer (file)
 
-Display raster data from a local file.
+Create and use a raster layer made from a local raster file.
 
-![](RasterLayerFile.png)
+![Image of apply raster layer file](RasterLayerFile.png)
 
-## How it works
+## Use case
+
+Rasters can be digital aerial photographs, imagery from satellites, digital pictures, or even scanned maps. An end-user will frequently need to import raster files acquired through various data-collection methods into their map to view and analyze the data.
+
+## How to use the sample
 
-To add a`RasterLayer` as an operational layer from a local raster file:
+When the sample starts, a raster will be loaded from a file and displayed in the map view.
+
+## How it works
 
-1. Create a`Raster` from a raster file
-2. Create a`RasterLayer` from the raster
-3. Add it as an operational layer with `map.getOperationalLayers().add(rasterLayer)`
+1. Create a `Raster` from a raster file.
+2. Create a `RasterLayer` from the raster.
+3. Add it as an operational layer with `map.getOperationalLayers().add(rasterLayer)`.
 
 ## Relevant API
 
-* ArcGISMap
-* Basemap
-* MapView
 * Raster
 * RasterLayer
+
+## Additional information
+
+See the topic [What is raster data?](http://desktop.arcgis.com/en/arcmap/10.3/manage-data/raster-and-images/what-is-raster-data.htm) in the *ArcMap* documentation for more information about raster images.
+
+## Tags
+
+data, image, import, layer, raster, visualization
```

Comment:
Same with this PR, we need to make sure there and empty newline at the end of each file